framework/web/JsonResponseFormatter.php

```diff
@@ -101,24 +124,20 @@ protected function formatJson($response)
      */
     protected function formatJsonp($response)
     {
-        $this->setContentType($response, 'application/javascript; charset=UTF-8');
-        if (is_array($response->data) && isset($response->data['data'], $response->data['callback'])) {
-            $response->content = sprintf('%s(%s);', $response->data['callback'], Json::htmlEncode($response->data['data']));
+        if (is_array($response->data)
+            && isset($response->data['data'], $response->data['callback'])
+        ) {
+            $response->content = sprintf(
+                '%s(%s);',
+                $response->data['callback'],
+                Json::htmlEncode($response->data['data'])
+            );
         } elseif ($response->data !== null) {
             $response->content = '';
-            Yii::warning("The 'jsonp' response requires that the data be an array consisting of both 'data' and 'callback' elements.", __METHOD__);
```

Comment:
Is this code style refactoring a subject of this patch?

Reply:
Yes. It is OK.